nodejs/awsx/examples/api/index.ts

```diff
@@ -67,23 +55,103 @@ const api = new awsx.apigateway.API("myapi", {
         path: "/b",
         method: "GET",
         eventHandler: lambda,
+        authorizers: [{
+            authorizerName: "prettyAuthorizer",
+            parameterName: "auth",
+            parameterLocation: "query",
+            authType: "custom",
+            authorizer: {
+                type: "request",
+                authorizer: async (event) => {
+                    console.log("Received event:", JSON.stringify(event, null, 2));
+                    const policy: awsx.apigateway.AuthorizerResponse = {
+                        principalId: "me",
```

Comment:
is "me" actually something supposed to be used?

Reply:
I'll add a comment here. in theory if you were getting a user from a DB or something that would be passed back as the principalId for the Lambda /backend to then use.